Providers/OpenEBS/Installers/Operator/0.7/ansible/openebs.yaml

```diff
@@ -0,0 +1,31 @@
+# TODO 
+# Apply openebs operator and openebs storage class
+
+- hosts: localhost
+  connection: local
+
+  vars_files:
+  - vars.yaml
+
+  tasks:
+    - name: Applying openebs operator
+      shell: "{{ kubeapply }} apply -f {{ openebs_operator_link }}" 
+      args:
+        executable: /bin/bash
+
+    - name: Applying openebs storageclass 
+      shell: "{{ kubeapply }} apply -f {{ openebs_storageclass_link }}" 
+      args:
+        executable: /bin/bash
+
+    - name: Applying openebs storagepoolclaim 
+      shell: "{{ kubeapply }} apply -f {{ openebs_storagepoolclaim_link }}" 
+      args:
+        executable: /bin/bash
+

```

Comment:
Can we add the validations for all the 0.7 components installed as part of operator, and also the sc, sp?

Reply:
yes will be adding need to discuss on that .